cmd/ui/src/views/Users/CreateUserForm.tsx

```diff
@@ -344,18 +344,14 @@ const CreateUserForm: React.FC<{
                             </FormHelperText>
                         )}
                         <Button
```

Comment:
Pretty minor nit, now that these buttons pulled in from Doodle are fully outlined, they seem crammed against the edge in all our modal dialogs and might benefit from slightly larger bottom & right margins:
<img width="633" alt="Screenshot 2024-07-22 at 1 14 26 PM" src="https://github.com/user-attachments/assets/fb03c5f1-45d8-4f3e-8ad3-dc8a3e371c2f">

Reply:
Good eye and agreed, updated modal actions container to have spacing that matches content/header